docs/advanced-table-feature.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Advanced table feature"
+slug: /advanced-table-feature
+date: 2023-12-19
+keyword: partitioning bucket distribution sort order
+license: Copyright 2023 Datastrato Pvt Ltd. This software is licensed under the Apache License version 2.
+---
+
+#### Partitioned table
+
+Currently, Gravitino supports the following partitioning strategies:
+
+:::note
+The `score`, `dt` and `city` are the field names in the table.
+:::
+
+| Partitioning strategy | Json                                                | Java                           | SQL syntax                 | Description                                                                                                                 |
+|-----------------------|-----------------------------------------------------|--------------------------------|----------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| Identity              | `{"strategy":"identity","fieldName":["score"]}`     | `Transforms.identity("score")` | `PARTITION BY score`       | Partition by a field or reference                                                                                           |
```

Comment:
`Identity` is also a function name.

Reply:
> Identity is also a function name.


Yes, if viewed strictly.

Reply:
So this part is redundant. This diagaram will make users confusing.

Reply:
Okay, I will remove it.